content/soarian/dstu2/infrastructure/documentReference.md

```diff
@@ -0,0 +1,97 @@
+---
+title: DocumentReference | Soarian DSTU 2 API
+---
+  
+# DocumentReference
+
+* TOC
+{:toc}
+
+## Overview
+
+For the DocumentReference resource query, the API returns one or more links to Summary of Episode Note (SOEN) consolidated clinical document architecture (C-CDA) documents stored in the Soarian Clinicals C-CDA archive for the subject patient.  If more than one C-CDA link is returned, they are sequenced in reverse chronological order by C-CDA creation date for user selection and display.  Note that each instance of a C-CDA is a unique document. Therefore, if multiple C-CDAs are created for a patient during an encounter where the creation date qualifies them for the query response, all qualified records are returned.  The C-CDA document archive may contain documents created by Soarian or by external systems; all qualifying (SOEN type by create date) records are returned regardless of their source.
+
+## Terminology Bindings
+
+<%= terminology_table(:soarian_document_reference, :dstu2) %>
+
+## Custom Extensions
+
+The [bundle extension] can be returned, see possible codes in the [errors section] below.
+
+## Search
+
+	GET /DocumentReference?:parameters
+
+_Implementation Notes_
+
+* The API returns the most current DocumentReference list.
+
+* The Narrative includes additional information pertaining to the document reference, and should be shown to the user to ensure completeness of clinical content and context.
+  
+
+### Parameters
+
+|Name |Required? | Type | Description
+| --- | --- | --- | --- |
+| patient | Y | [reference](http://hl7.org/fhir/DSTU2/search.html#reference) | The patient identifier provided in a pre-requisite authorization step. |
+| date | N | [date](http://hl7.org/fhir/DSTU2/search.html#date) as adjusted per implementation notes | Null or specific date or a date range. |
+| attachment | N |URL encoded String| attachmentId is retrived from attachment section base url | 
```

Comment:
Attachment is the value that is returned in the element “content.attachement.url” as part of the response to the initial “DocumentReference” request. We will update the same in the description.